urdf/urdfdom_compatibility.h.in

```diff
@@ -41,7 +41,7 @@
 #define URDFDOM_HEADERS_MINOR_VERSION @URDFDOM_HEADERS_MINOR_VERSION@
 #define URDFDOM_HEADERS_REVISION_VERSION @URDFDOM_HEADERS_REVISION_VERSION@
 
-// for Wily: maintain compatibility between urdfdom 0.3 and 0.4 (definining SharedPtr types)
+// Define shared pointers for urdfdom versions less than 0.4
```

Comment:
Nit, but I'd update this comment to be a little more descriptive of the archeology we did.  Something like:

```
// For urdfdom_headers < 0.4, need to define all of these.  For urdfdom_headers == 0.4, still need this to define ModelSharedPtr and ModelInterfaceSharedPtr, and doesn't hurt to re-define the rest
```

Or something like that.